packages/fuels-test-helpers/src/node.rs

```diff
@@ -247,12 +318,13 @@ pub async fn new_fuel_node(
     coins: Vec<(UtxoId, Coin)>,
     consensus_parameters_config: Option<ConsensusParameters>,
     config: Config,
+    input_messages: Option<Vec<Message>>,
```

Comment:
Again, why `input_messages` instead of just `messages`? And why not put it right after `coins` like you did with `get_node_config_json`